packages/angular/migrations.spec.ts

```diff
@@ -3,9 +3,11 @@ import json = require('./migrations.json');
 
 describe('Angular migrations', () => {
   it('should have valid paths', () => {
-    Object.values(json.schematics).forEach((m) => {
+    Object.values(json.schematics).forEach((m: any) => {
```

Comment:
A heads up more than anything. There's a PR that changes this json to `generators` instead of `schematics`.

https://github.com/nrwl/nx/pull/16291